csfieldguide/static/interactives/pixel-viewer/js/pixel-viewer.js

```diff
@@ -151,7 +151,7 @@ function setUpMode(){
 
```

Comment:
Perhaps add a warning or something to say the 'add noise' button will freeze the interactive while it does the calculations, and that any actions you make during that time will be applied retroactively all at once